pkg/registry/generic/etcd/etcd.go

```diff
@@ -54,6 +54,8 @@ import (
 // logic specific to the API.
 //
 // TODO: make the default exposed methods exactly match a generic RESTStorage
+// TODO: because all aspects of etcd have been removed it should really
```

Comment:
+1 from me.

However, I remember @smarterclayton wasn't convinced about it at some point (although I'm not 100% it was about this particular class). @smarterclayton - thought about it?